### PR TITLE
Article page

### PR DESCRIPTION
**Pull request by ferbaco86**
- New York Times Newspaper article webpage clone
- This branch 'article-page' includes the first working version of the project

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,687 @@
...
+  padding-left: 10px;
+  display: flex;
+  align-items: center;
+}
+
+.ul-subscribe {
+  border-left: 1px solid rgb(226, 226, 226);
+  margin-bottom: -5px;
+
+}
```
[the remark this answers is not in the document]
Comment (FrancoRosa) [reply css/style.css:599]: Empty space deleted

**Review by mosesogwo — CHANGES_REQUESTED** [review]
**Hi @ferbaco86 & @... You have done well on this project:+1:. Just one change is required:**

- The New York Times logo should be a link.

**Kindly make this change and submit another code review request.**_

**Any question? You can reach me on Slack (Moses Ogwo)**

**Cheers:stars:**

### Status: Changes Requested.

**Comment by kobiyoyo** [comment]
## Pull Request under review, by Adama Chubiyojo Desmond (TSE)

**Review by kobiyoyo — APPROVED** [review]
# Project Approved :thumbsup:
## Hello Team,  You have done a great job with this project. :clap:

**Pull request merged**